bodhi/server/models.py

```diff
@@ -3339,7 +3365,7 @@ def met_testing_requirements(self):
         if min_num_days:
             if not self.meets_testing_requirements:
                 return False
-        else:
+        elif not self.autotime:
```

Comment:
I still think we should not use the new boolean to determine whether an update has met testing requirements - we need to keep the concept of requirements and automation machinery separate.

Reply:
Yes I think if we get https://github.com/fedora-infra/bodhi/pull/3158 merged before this one then we don't need this change.
Otherwise I can manage this special case in the script instead of here.